docs/build.md

```diff
@@ -0,0 +1,61 @@
+# Installation and Building
+
+## Prerequisites
+
+In order to build Wallet from the source, first make sure to install the following:
+
+- [Go 1.17](https://go.dev/doc/install)
+- [Npm 8](https://docs.npmjs.com/cli/v8/configuring-npm/install)
+- [Node 17](https://nodejs.org/)
+- GitHub packages setup: you will need to authenticate to GitHub packages with your [personal token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line#creating-a-token).
+- Configuring `npm` for use with GitHub Packages `echo "//npm.pkg.github.com/:_authToken=${PERSONAL_TOKEN}" > ~/.npmrc`
+- [Docker](https://docs.docker.com/get-docker/) (**Note**: make sure to set your `Docker` to use `Docker-Compose V1`)
+- [Docker-Compose V1](https://docs.docker.com/compose/install/)
+- Make
+
+## Test Keys
+
+In addition, run (`make generate-test-keys`) to generate tls keys and import `test/fixtures/keys/tls/ec-cacert.pem` in your machine's cert chain (e.g. Keychain on Mac).
+
+## Connections
+
+You also need to modify your hosts file (`/etc/hosts` on \*NIX) to add the following lines, to allow few of the bdd test containers to be connected to externally.
```

Comment:
Something like this might be more clear. 

This etc/host file is part of Docker networking mechanism which is primarily used to establish communication between Docker containers and the outside world via the host machine where the Docker daemon is running.